src/apiCalls.js

```diff
@@ -1,9 +1,17 @@
 export const getAllMovies = () => {
     return fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
-     .then(response => response.json())
+     .then(response => checkForErrors(response))
 }
 
 export const fetchSingleMoviePoster = (id) => {
   return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
-   .then(response => response.json())
+   .then(response => checkForErrors(response))
+}
+
+export const checkForErrors = (response) => {
+  if (!response.ok) {
+    throw new Error(`${response.status} - something went wrong. ${response.statusText}`);
+  }
+  console.log("RESPONSE", response)
+  return response.json()
```

Comment:
Nice! I really like this strategy for handling a URL error. We can style this error page a little more later :)